Assets/Scripts/Player/PlayerResources.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerResources : MonoBehaviour
+{
+    
+    public PlayerResource Health;
+
+}
```

Comment:
Recommend initializing each PlayerResource here, as the ScriptableObject's state will not reset between plays (at least from the Unity Editor).

```suggestion
public class PlayerResources : MonoBehaviour
{

    public PlayerResource Health;

    void Awake() {
        Health.Init();
    }

}
```

Reply:
The `OnEnable` approach might work too, I've never used a ScriptableObject that way and would be curious to hear about your experience with this method.